src/client/src/containers/Details/index.tsx

```diff
@@ -1,6 +1,12 @@
 import classnames from "classnames";
 import * as React from "react";
 import ReactMarkdown from "react-markdown";
+import { connect } from "react-redux";
+import { ThunkDispatch } from "redux-thunk";
+
+import { AppState } from "../../reducers";
+import RootAction from "../../actions/ActionType";
+import * as ModalActions from "../../actions/modalActions/modalActions";
```

Comment:
Small question. Do you really need to import * for this? 
Looks like you're only using openRedirectModalAction. Couldn't you do import { openRedirectModalAction } 
instead? Is there a reason why you're doing it like this rather than what I mention?

Reply:
Good point. Changed it.